docs/config/README.md

```diff
@@ -810,11 +810,12 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 ### Options
 
-| Variable   | Default       | Description                                         |
-| ---------- | ------------- | --------------------------------------------------- |
-| `symbol`   | `"☸ "`        | The symbol used before displaying the Cluster info. |
-| `style`    | `"bold blue"` | The style for the module.                           |
-| `disabled` | `true`        | Disables the `kubernetes` module                    |
+| Variable         | Default       | Description                                         |
+| ---------------- | ------------- | --------------------------------------------------- |
+| `symbol`         | `"☸ "`        | The symbol used before displaying the Cluster info. |
+| `style`          | `"bold blue"` | The style for the module.                           |
+| `disabled`       | `true`        | Disables the `kubernetes` module                    |
+| `context_aliases` |               | Table of context aliases to display                 |
```

Comment:
We normally put the `style` and `disabled` variables at the bottom of the table.

```suggestion
| `context_aliases` |               | Table of context aliases to display                 |
| `style`          | `"bold blue"` | The style for the module.                           |
| `disabled`       | `true`        | Disables the `kubernetes` module                    |
```